fpga/lib/core_connection.cpp

```diff
@@ -0,0 +1,71 @@
+/* Memorygraph-connection to a fpga ip core.
```

Comment:
can you add a bit more description? This only supports platform. Explain why this class is required.